LTTngDriver/Program.cs

```diff
@@ -210,6 +242,14 @@ public bool Run()
             }
         }
 
+        public static bool IsList(object o)
+        {
+            if (o == null) return false;
+            return o is IList &&
+                   o.GetType().IsGenericType &&
+                   o.GetType().GetGenericTypeDefinition().IsAssignableFrom(typeof(List<>));
```

Comment:
"IList<>"?

Reply:
Yea this does seem a bit odd but I tried your way /w  IsAssignableFrom(typeof(IList<>) and it broke the check. This is the answer from StackOverflow and seems to work well.